index.html

```diff
@@ -0,0 +1,113 @@
+  <div class="container">
+    <h1 id="quiz">The Quiz Slide-2</h1>
+    <p id="questions">How do you write "Hello World" in an alert box?</p>
+    
+    <button type="button" class="btn btn-secondary btn-lg btn-block">msgBox('hello world')</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">msg('hello world')</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">alert('hello world')</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">alertBox('hello world')</button>
+  </div>
+</div>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 id="quiz">The Quiz Slide-3</h1>
+    <p id="questions">How do you create a function in JavaScript?</p>
+    
+    <button type="button" class="btn btn-secondary btn-lg btn-block">function:myfunction</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">function = myFunction</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">function myFunction()</button>
+    
+  </div>
+</div>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 id="quiz">The Quiz slide-3</h1>
+    <p id="questions">How to write an IF statement for executing some code if "i" is NOT equal to 5?</p>
+    
+    <button type="button" class="btn btn-secondary btn-lg btn-block">if i <> 5</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">if (i ! =5)</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">if (i <> 5)</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">if i = ! 5 then</button>
+  </div>
+</div>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 id="quiz">The Quiz slide-4</h1>
+    <p id="questions">What is the correct way to write a JavaScript array?</p>
+    
+    <button type="button" class="btn btn-secondary btn-lg btn-block">var colors = (1:'red',2:'green',3:'blue'</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">var colors = ['red','blue','green']</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">var colors ='red', 'blue', 'green'</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">var colors =1 = ('red'), 2 = ('green'), 3 = ('blue')</button>
+  </div>
+</div>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 id="quiz">The Quiz slide-5</h1>
+    <p id="questions">Which event occurs when the user clicks on an HTML element?</p>
+    
+    <button type="button" class="btn btn-secondary btn-lg btn-block">onclick</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">onmouseover</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">onmouseclick</button>
+    <button type="button" class="btn btn-secondary btn-lg btn-block">onchange</button>
+  </div>
+</div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
here is where i would either write a script for handling the clicks on the questions and also to handle the questions and the answer clicked